src/gt4py/gtscript.py

```diff
@@ -450,7 +487,7 @@ def computation(order):
     return _ComputationContextManager()
 
 
-def interval(start, end):
+def interval(*args):
```

Comment:
Unrelated fix: linter complains on `interval(...)` that there is not value for `end` without this change.

Reply:
Just a side note here: it the linting problem here is mypy, this could be solved using `overload`
https://mypy.readthedocs.io/en/stable/more_types.html#function-overloading
https://docs.python.org/3/library/typing.html#typing.overload

Reply:
Oh, I like the `@overload` more. I didn't know that existed! I'll make that change.

Reply:
I can't seem to get this to work. I keep getting the error:

```
error: An overloaded function outside a stub file must have an implementation
```